src/DbConn/cbio_db.py

```diff
@@ -0,0 +1,126 @@
+from bravado.client import SwaggerClient
+import pandas as pd 
+
+
+class CBioDb:
+    def __init__(self) -> None:
+        self.conn = SwaggerClient.from_url('https://www.cbioportal.org/api/v2/api-docs',
+                                           config={"validate_requests":False,"validate_responses":False,"validate_swagger_spec":False})
+        self._all_study_ids = None 
+        self._all_studies = None 
+        self._all_fields = None 
+
+    @property
+    def all_studies(self):
+        if self._all_study_ids is None:
+            return self.conn.Studies.getAllStudiesUsingGET().result()
+
+    @property
+    def all_fields(self):
+        if self._all_fields is None:
+            return dir(self.all_studies[0])
+            
+    @property
+    def all_study_ids(self):
+        if self._all_study_ids is None:
+            study_ids = [x.studyId for x in self.all_studies]
+            return study_ids
+            
+    def get_study(self, study_id):
+        return self.conn.Studies.getStudyUsingGET(studyId=study_id).result()
+    
+    def get_clinical_data(self, study_id):
+        return self.conn.Clinical_Data.getAllClinicalDataInStudyUsingGET(studyId=study_id).result()
+    
+    def get_case_lists(self, study_id):
+        return self.conn.Case_Lists.getAllCaseListsInStudyUsingGET(studyId=study_id).result()
+    
+    def get_case_list(self, study_id, case_list_id):
+        return self.conn.Case_Lists.getCaseListUsingGET(studyId=study_id, caseListId=case_list_id).result()
+    
+    def get_case_list_data(self, study_id, case_list_id):
+        return self.conn.Case_Lists.getAllCasesInCaseListUsingGET(studyId=study_id, caseListId=case_list_id).result()
+    
+    def get_molecular_profiles(self, study_id):
+        return self.conn.Molecular_Profiles.getAllMolecularProfilesInStudyUsingGET(studyId=study_id).result()
+    
+    def get_molecular_profile(self, study_id, molecular_profile_id):
+        return self.conn.Molecular_Profiles.getMolecularProfileUsingGET(studyId=study_id, molecularProfileId=molecular_profile_id).result()
+    
+    def get_molecular_profile_data(self, study_id, molecular_profile_id):
+        return self.conn.Molecular_Profiles.getAllDataInMolecularProfileUsingGET(studyId=study_id, molecularProfileId=molecular_profile_id).result()
+    
+    def get_genes(self, study_id):
+        return self.conn.Genes.getAllGenesInStudyUsingGET(studyId=study_id).result()
+    
+    def get_gene(self, study_id, gene_id):
+        return self.conn.Genes.getGeneUsingGET(studyId=study_id, geneId=gene_id).result()
+    
+    def get_gene_data(self, study_id, gene_id):
+        return self.conn.Genes.getAllGeneDataUsingGET(studyId=study_id, geneId=gene_id).result()
+    
+    def get_all_patients(self, study_id):
+        return self.conn.Patients.getAllPatientsInStudyUsingGET(studyId=study_id).result()
+    
+    def get_data(self, method_name, **kwargs):
```

Comment:
since this is the only method that is calling the api that is used, I would remove all the others - also, did we come to a determination whether we need api/db calling code at all for this project? if not, I would remove it altogether

Reply:
I think for now we aren't really using this API - could be useful in the future. I will remove this from the pull request.